core/src/main/java/org/bitcoinj/evolution/SimplifiedMasternodeListManager.java

```diff
@@ -1097,7 +1106,7 @@ public void run() {
                     } catch (IOException x) {
 
                     } catch (BlockStoreException x) {
-
+                        throw new RuntimeException(x);
                     }
                 }
             }
```

Comment:
This class is getting big. Not sure if it's the same one that I noted the previous time

Reply:
This is the same file.  Last time you noted this it was 1600 lines, now it is about 1100.

In the next PR, I can refactor further.  Perhaps the bootstrap functionality can go to its own class.

Reply:
Oh ok, great work on refactoring so far.